index.html

```diff
@@ -0,0 +1,282 @@
+                  born and I account will give you a complete account of the
+                  system
+                </div>
+                <div class="booknow2_button">
+                  <button class="booknow2__button" type="submit">book now</button>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="whattouristsay">
+        <h2 class="whattouristsay__title whattouristsay_title">What Our Tourists Say</h2>
+        <div class="container">
+          <div class="row">
+            <div class="col-4">
+              <img src="img/ReviewPhoto.png" alt="Tolik v Karpatah" />
+            </div>
+            <div class="col-8">
+              <div class="whattouristsay__article">
+                <h4 class="whattouristsay__name whattouristsay_name">Sam Sumon</h4>
+                 <p class="whattouristsay__text whattouristsay_text"> <svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+                  <path d="M18.125 10H15V7.5C15 6.12109 16.1211 5 17.5 5H17.8125C18.332 5 18.75 4.58203 18.75 4.0625V2.1875C18.75 1.66797 18.332 1.25 17.8125 1.25H17.5C14.0469 1.25 11.25 4.04688 11.25 7.5V16.875C11.25 17.9102 12.0898 18.75 13.125 18.75H18.125C19.1602 18.75 20 17.9102 20 16.875V11.875C20 10.8398 19.1602 10 18.125 10ZM6.875 10H3.75V7.5C3.75 6.12109 4.87109 5 6.25 5H6.5625C7.08203 5 7.5 4.58203 7.5 4.0625V2.1875C7.5 1.66797 7.08203 1.25 6.5625 1.25H6.25C2.79688 1.25 0 4.04688 0 7.5V16.875C0 17.9102 0.839844 18.75 1.875 18.75H6.875C7.91016 18.75 8.75 17.9102 8.75 16.875V11.875C8.75 10.8398 7.91016 10 6.875 10Z" fill="#5373FD"/>
+                  </svg>
+                  
+                 But I must explain to you how all this mistaken idea of
+                  account to denouncinpleasure and praising pain But I must
+                  explain to you how all this mistaken idea of account to nai
+                  denouncing pleasure and praising pain was account born and I
+                  account will give you a complete account of the system was
+                  account born and I account will give you a nai the complete
+                  account of the system
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <footer class="footer">
+        <h2 class="footer__title footer_title">Bhromaon</h2>
+        <p class="footer__subtitle footer_subtitle">Safe Travel Anywhere</p>
+        <div class="footer__images">
+          <svg class="footer__svg" width="20" height="32" viewBox="0 0 40 64" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <path d="M34.8925 36L36.67 24.4175H25.5562V16.9013C25.5562 13.7325 27.1087 10.6438 32.0862 10.6438H37.1387V0.7825C37.1387 0.7825 32.5537 0 28.17 0C19.0175 0 13.035 5.5475 13.035 15.59V24.4175H2.86124V36H13.035V64H25.5562V36H34.8925Z" fill="white"/>
+            </svg>
```

Comment:
```suggestion
          <svg class="footer__svg" width="20" height="32" viewBox="0 0 40 64" fill="white" xmlns="http://www.w3.org/2000/svg">
            <path d="M34.8925 36L36.67 24.4175H25.5562V16.9013C25.5562 13.7325 27.1087 10.6438 32.0862 10.6438H37.1387V0.7825C37.1387 0.7825 32.5537 0 28.17 0C19.0175 0 13.035 5.5475 13.035 15.59V24.4175H2.86124V36H13.035V64H25.5562V36H34.8925Z" />
            </svg>
```
чтоб легче было менять заливку через CSS